js/gameover.js

```diff
@@ -1,19 +0,0 @@
-function gameOver() {
-  background("red");
-  image(imgExplosion, player[0].x, player[0].y, 90, 90);
-  noLoop();
-
-  clock.stop();
-  setTimeout(() => {
-    soundtrack.stop();
-    document.getElementById("game-screen").classList.toggle("hidden");
-    document.getElementById("gameover").classList.toggle("hidden");
-    let displayClock = document.getElementById("gameover");
-    displayClock.innerHTML = clock.split();
-  }, 1000);
-
-  setTimeout(() => {
-    document.location.reload();
```

Comment:
is this an auto-restart after 5 s? 
why do you do this with a reload? 

instead you could write a function that resets game state -> than it wouldn't reload the page, but instead just reset the variables within the script and then go from there. 

I like the solution though, it's creative ^^